profiles/desktop.py

```diff
@@ -4,6 +4,10 @@
 
 is_top_level_profile = True
 
+# New way of defining packages for a profile, which is iterable and can be used out side
+# of the profile to get a list of "what packages will be installed".
+__packages__ = ['nano', 'openssh', 'htop', 'wget', 'iwd', 'wireless_tools', 'wpa_supplicant', 'smartmontools']
```

Comment:
I wonder if we really need the wireless tools.
They are pulled in if they opt in to configure the network from the option "Copy configuration from the ISO" I believe.

Reply:
Honestly, maybe not, but I will say that I wasn't super happy that I had no way of connecting to a network on a new install and had to boot off the ISO and do a whole new installation (I mean I could have plugged in a USB ethernet adapter but that's another story)

Reply:
Anything that leaves the system in a state where a user can't easily use it without a reinstall is high-priority in my mind.

Reply:
> Anything that leaves the system in a state where a user can't easily use it without a reinstall is high-priority in my mind.

Agreed. But this is what the warning in the network-step is for I think.
I don't mind them being in desktop tho, seeing as it's optional anyway.
But might be worth re-visiting later *(and if so slim PR's like we did with KDE and Gnome)*.

I'll leave them in for now since it doesn't bloat the guided which is the main concern : )